### PR TITLE
Adds notes on conda on ARCHER2

### PR DESCRIPTION
**Pull request by aturner-epcc**

### CODE REVIEW

File: docs/user-guide/python.md [hunk 1]
```diff
@@ -124,6 +124,117 @@ ensuring that the python packages will be gathered from the local virtual enviro
...
+of ARCHER2. In particular, you should be careful of:
+
+- Where you install conda on ARCHER2
+- Conda additions to shell configuration files such as `.bashrc`
+
+We cover each of these points in more detail below.
+
+### Conda install location
+
+If you only need to use the files and executables from your conda installaiton on the login and data analysis nodes
```
Comment (juanfrh) [inline docs/user-guide/python.md:141]: _installaiton_ misspelled.

File: docs/user-guide/python.md [hunk 2]
```diff
@@ -124,6 +124,117 @@ ensuring that the python packages will be gathered from the local virtual enviro
...
+```
+
+Once you have the container image, you can run scripts in it with a command like:
+
+```
+singularity exec -B $PWD anaconda3.sif python my_script.py
+```
+
+As the container image is a single large file, you end up doing a single large read from the work file system rather
+than lots of small reads of individual Python files,, this improves the performance of Python and reduces the
```
Comment (juanfrh) [inline docs/user-guide/python.md:171]: double comma `,,`

File: docs/user-guide/python.md [hunk 3]
```diff
@@ -124,6 +124,117 @@ ensuring that the python packages will be gathered from the local virtual enviro
...
+you login to ARCHER2, the conda environment will try to initialise by reading lots of files
+within the conda installation. This approach was designed for the case where a user has installed
+conda on their personal device and so is the only user of the file system. For shared file systems
+such as those on ARCHER2, this places a large load on the file system and will lead to you seeing 
+slow login times and slow response from your command line on ARCHER2. It will also lead to degraded
+read/write performance from the work file systems for you and other users so should be avoided at 
+all costs.
+
+If you have previously installed a conda distribution and answered "yes" to the question about
+adding the initialisation to shell configuraiton files, you should edit your `~/.bashrc` file
```
Comment (juanfrh) [inline docs/user-guide/python.md:216]: _configuraiton_ misspelled.

**Review by juanfrh — COMMENTED** [review]
Thanks for writing this piece of documentation. I tested the commands and work as expected. I've just highlighted some minor typos.

File: docs/user-guide/python.md (hunk 1 above)
Comment (aturner-epcc) [reply docs/user-guide/python.md]: Fixed

File: docs/user-guide/python.md (hunk 3 above)
Comment (aturner-epcc) [reply docs/user-guide/python.md]: Fixed

File: docs/user-guide/python.md (hunk 2 above)
Comment (aturner-epcc) [reply docs/user-guide/python.md]: Fixed

**Comment by aturner-epcc** [comment]
Thanks @juanfrh I think I fixed all the issues now

**Review by juanfrh — APPROVED** [review]
LGTM, thanks Andy.

**Pull request merged**